File: PostRequest2/app/src/main/java/com/example/postrequest/SecondActivity.java
package com.example.postrequest;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class SecondActivity extends AppCompatActivity {
    private TextView tvTitle, tvRequest, tvPasswd, tvEmail;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        tvEmail = findViewById(R.id.tvEmail);
        tvPasswd = findViewById(R.id.tvPasswd);
        tvRequest = findViewById(R.id.tvRequestType);
        tvTitle = findViewById(R.id.tvTitle);
        Intent intent = getIntent();

        int id = intent.getIntExtra("id", 0);
        String email = intent.getStringExtra("email");
        String title = intent.getStringExtra("title");
        String request = intent.getStringExtra("request");

        tvTitle.setText("Title: " + title);
        tvEmail.setText("Email:" + email);
        tvRequest.setText("Request Type: " + request);
        tvPasswd.setText("ID: " + id);

    }
}

File: Kotlin Projects/PostRequest_Kotlin/app/src/main/java/com/example/postrequest_kotlin/SecondActivity.kt
package com.example.postrequest_kotlin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_second.*

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        val intent = intent

        val id = intent.getIntExtra("id", 0)
        val email = intent.getStringExtra("email")
        val title = intent.getStringExtra("title")
        val request = intent.getStringExtra("request")

        tvTitle.text = ("Title: $title")
        tvEmail.text = ("Email:$email")
        tvRequestType.text = ("Request Type: $request")
        tvPasswd.text = ("ID: $id")
    }
}